libsolidity/interface/OptimiserSettings.h

```diff
@@ -47,7 +47,7 @@ struct OptimiserSettings
 			"gvif"                     // Run full inliner
 			"CTUcarrLsTOtfDncarrIulc"  // SSA plus simplify
 		"]"
-		"jmuljuljul VcTOcul jmul";     // Make source short and pretty
+		"jmuljuljul VcTOcul jmulz";     // Make source short and pretty
```

Comment:
I would do it before the constant optimizer and the rematerializer.

Reply:
Actually, moving it anywhere further up will increase gas cost.
The reason is probably that the rematerializer actually cannot rematerialize ``returndatasize`` since in general it's not movable.
So I'd just keep it at the very end for now. Maybe things will be different, once we make side-effects more fine-grained.